assets/onboarding/features/index.jsx

```diff
@@ -24,34 +25,40 @@ const Features = () => {
 	} = useSetupWizardStep( 'features' );
 	const features = stepData.options;
 
+	useEffect( () => {
```

Comment:
Here I think almost everything is related to the previous commented changes, right? With an exception for the `logEvent `